README.md

```diff
@@ -121,8 +123,12 @@ keys or permissions. You just need to set the server url in `config/plugins/sour
 
 ## Discord
 
-The Discord plugin loads Discord servers, and mints Cred on Discord reactions.
-For instructions on configuring the Discord plugin, see the [Discord plugin page](https://sourcecred.io/docs/beta/plugins/discord/#configuration) in the SourceCred documentation. 
+The Discord plugin loads Discord servers, and mints Cred on Discord reactions. In order for SourceCred to
+access your Discord server, you need to generate a "bot token" and paste it in the `.env` file as well as adding it
```

Comment:
so I just paste the discord token directly into the .env file? I'm pretty sure you need a line saying `SOURCECRED_DISCORD_TOKEN=...`. Please be more specific. I suggest referencing the .env.example file.